sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -482,6 +495,65 @@ export class AssessmentController {
         ? output[outputName] + award.total
         : award.total;
     });
+    ftProgramYearContributionTotal?.forEach((ftContributionTotal) => {
+      const outputName = `programYearTotal${ftContributionTotal.contribution}`;
+      output[outputName] = output[outputName]
+        ? output[outputName] + ftContributionTotal.total
+        : ftContributionTotal.total;
+    });
+  }
+
+  /**
+   * Get the promises of the program year awards totals for part time and
```

Comment:
This method should not return multiple promises expected to be awaited by the consumer.
Both promises can be awaited in parallel and their results can be returned instead.